documentation/tutorial.md

```diff
@@ -68,3 +68,145 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploying Charmed MongoDB:
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Accessing MongoDB:
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. It is not recommended that you directly interface with the admin user in a production environment. In a production environment [create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead.*
```

Comment:
Hm, sections are not numbered so can section 4 be found at a glance?

Also, the purpose of a tutorial, as I understood from Daniele, is to remove as much ambiguity and choices as possible. Adding the note where we explain that this is not a production setup is good, but I don't see the use on having a link to a different section. I think the goal is to minimize branching attention to things, and just focus on completing the tutorial, step by step.

Reply:
ooh, great point! What if instead of branching we state:
```
Later in [the section covering Relations](#relations) we will cover how to access MongoDB without the admin user.
```

(note section \#relations  will be added in a following PR). Thoughts @zmraul and @deusebio

Reply:
I could go either way, @deusebio what do you think?

Reply:
I would drop the link, while retaining the reference that this will be covered later. It would be best to require reader to read through, without skipping parts (as a link would nudge)